### PR TITLE
ref: Remove some index files

### PR DESCRIPTION
**Pull request by robertpin**
Fix imports

### CODE REVIEW

File: react/features/recording/components/LiveStream/native/StreamKeyPicker.js [hunk 1]
```diff
@@ -43,10 +43,10 @@ type Props = {
 
 type State = {
 
-     /**
-      * The key of the currently selected stream.
-      */
-     streamKey: ?string
+    /**
```
Comment (Calinteodor) [inline react/features/recording/components/LiveStream/native/StreamKeyPicker.js:46]: Unrelated change.

File: react/features/subtitles/components/Captions.web.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { connect } from 'react-redux';
 
 import { getLocalParticipant } from '../../base/participants';
 import { getLargeVideoParticipant } from '../../large-video/functions';
-import { isLayoutTileView } from '../../video-layout';
+import { isLayoutTileView } from '../../video-layout/functions.any';
```
Comment (Calinteodor) [inline react/features/subtitles/components/Captions.web.js:8]: functions.web

File: react/features/toolbox/components/native/HangupMenu.tsx [hunk 3]
```diff
@@ -29,7 +27,7 @@ import { isInBreakoutRoom } from '../../../breakout-rooms/functions';
  */
 function HangupMenu() {
     const dispatch = useDispatch();
-    const _styles = useSelector(state => ColorSchemeRegistry.get(state, 'Toolbox'));
+    const _styles: any = useSelector(state => ColorSchemeRegistry.get(state, 'Toolbox'));
```
Comment (Calinteodor) [inline react/features/toolbox/components/native/HangupMenu.tsx:30]: Unrelated change.

File: react/features/video-quality/components/VideoQualityLabel.web.js [hunk 4]
```diff
@@ -3,13 +3,13 @@
...
 
-import { openDialog } from '../../base/dialog';
+import { openDialog } from '../../base/dialog/actions';
 import { translate } from '../../base/i18n';
 import { IconPerformance } from '../../base/icons';
 import { Label } from '../../base/label';
 import { COLORS } from '../../base/label/constants';
 import Tooltip from '../../base/tooltip/components/Tooltip';
-import { shouldDisplayTileView } from '../../video-layout';
+import { shouldDisplayTileView } from '../../video-layout/functions';
```
Comment (Calinteodor) [inline react/features/video-quality/components/VideoQualityLabel.web.js:12]: functions.web

**Review by Calinteodor — CHANGES_REQUESTED** [review]

File: react/features/toolbox/components/native/HangupMenu.tsx (hunk 3 above)
Comment (robertpin) [reply react/features/toolbox/components/native/HangupMenu.tsx:30]: It's related. Because the `ColorSchemeRegistry` import is now fixed TS throws an error saying 'X' does not exists on type StyleType (X being the properties we use from `_styles`)
